routestest.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
-echo "hello world"
+echo "checking localhost:9001"
 
-curl -d "destination=STRING" -d  "message=STRING" http://localhost:9001/messages
 
-curl -d "fakeDescrption=STRING" -d  "message=STRING" http://localhost:9001/messages
+curl -X POST http://localhost:9001/messages  -d '{"destination" : "STRING", "message" : "STRING"}'  -H "content-type: application/json" -v
```

Comment:
Bravo 🥇 
I think you can do a bit better.
While these can test your routes you need manually to verify that they returned the expected result.

Do you think you can automate this? 
Search how you can save the result of a command to a variable
and search how if works in bash.